bounties/Code Sample For Airgapped Wallet.md

```diff
@@ -0,0 +1,115 @@
+<!-- Please update this title -->
+
+# Code Sample For Airgapped Wallet
+
+<!--
+Please add a high level category of work.
+Ex. Game Development, Open Source Tooling, Etc.
+-->
+
+**Category:** Technical Documentation
+
+<!--
+Recommend a total value amount for the bounty, in U.S. Dollars. The exact amount will be determined by the approving committee and may be higher or lower than this recommendation.
+-->
+
+**Amount:** $6,000
+
+**Proposed programming language:** Python
+
+## Overview
+
+<!--
+Please provide the context required to complete the bounty.
+
+Questions you should answer here:
+1. What is the high level explanation of this bounty? (1-3 sentences)
+2. What problem is this solving?
+3. What are the requirements for this solution?
+-->
+
+**Discussion:** https://github.com/XRPLBounties/Proposals/discussions/47 (Must Read for details)
+
+## Milestones
+
+<!--
+Please split the bounty into smaller milestones with individual awards in the following template.
+The first milestone should be the core functionality, while the rest can be useful add-ons.
+
+| # | High-Level Description | Details | Proposed Potential Award |
+| 1 | ... | ... | $ |
+
+(The proposed amounts from milestones should add up to the amount listed at the top of the bounty proposal)
+-->
+
+
+| # | High-Level Description | Details | Proposed Potential Award |
+| --- | --- | --- | --- |
+| 1 | Keypair/Wallet management system | Generate an XRPL account and store it securely with a password feature built-in using a cryptography algorithm | $1,500 |
```

Comment:
```suggestion
| 1 | Keypair/Wallet management system | Generate an XRPL account and store it securely with a password feature built-in using a cryptography algorithm | $1,000 |
```

Reply:
Can you provide a reasonable reasoning for this?

I suggested that prize for Milestone 1 as it required me to make the program available for use in both Linux and Windows & enable the use of cryptography to securely store the keypair internally (again: for educational use).

Reply:
In my opinion comparing to milestone 2 (same prize amount) this would require less amount of work - but open to hearing other engineer's estimation as well.

Reply:
I personally think $1,500 seems fine for the ask in this case

Reply:
Thank you.